torch/csrc/jit/passes/quantization.cpp

```diff
@@ -503,8 +503,56 @@ void InsertQuantDequantNodesForParam(
   }
 }
 
+void InsertQuantDequantNodesForParam(
+    script::Method& method,
+    const std::string& param_name,
+    const std::function<std::tuple<std::string, float, int>(float, float)>&
+        getQParamFunc,
+    at::ScalarType t) {
+  AT_ASSERT(getQParamFunc != nullptr);
+  auto params_to_insert_qdq = getQuantizableParamsofName(method, param_name);
+
+  for (auto& param_info : params_to_insert_qdq) {
+    // Fetch scale for input activation and weight to the quantizable
+    // node for which bias need to be quantized
+    Node* n = param_info.n;
+    // Check if this node has weight attr as input
+    size_t param_index = getParamIndexinOpArgs(n, std::string("weight"));
```

Comment:
`param_name` instead of hardcoded `std::string("weight")` here?

Reply:
For bias, we extract scale for weight and activation explicitly

Reply:
I understand what the code is doing, but I dont understand why. Why do we need to check for "weight" explicitly, even if `param_name` is "bias"? If that's intended behavior, it should be reflected in comments.

Reply:
updated comments